sssd_test_framework/roles/ad.py

```diff
@@ -161,6 +171,12 @@ def fqn(self, name: str) -> str:
         """
         return f"{name}@{self.domain}"
 
+    def dn(self, name: str) -> str:
+        """
+        Returns distinguished name in form of cn=name,naming_context.
+        """
+        return f"cn={name},{self.naming_context}"
```

Comment:
Could the CN here ever have to container a comma and could that break this in some way later?

Reply:
You're right, this does not account for different object types and different ou's, it should be 'cn={name},CN=users,{self.naming_context}'. I can't remember if I just saw FQN and thought this would be useful. Now that I'm thinking more about it, this kind of doesn't make sense. The dn should be available for any object, not just the role. 

role.dn (should just return the suffix)
user.dn (user dn, different ou)
computer.dn (another ou)

probably going to remove it.